core/src/main/java/com/netflix/conductor/service/utils/ServiceUtils.java

```diff
@@ -0,0 +1,89 @@
+package com.netflix.conductor.service.utils;
+
+import com.google.common.base.Preconditions;
+import com.netflix.conductor.core.execution.ApplicationException;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+public class ServiceUtils {
+
+    public static List<String> convertToSortedList(String sortStr) {
```

Comment:
Please add javadoc for public static method

Reply:
Seems like this method is used for splitting a string and does not sort like the name implies?